mitodl/mitxpro
# Add news and events carousel

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/mitxpro/issues/2108

#### What's this PR do?
Adds a NEWS/EVENTS section to HomePage.

#### How should this be manually tested?
* Go to cms
* Under HomePage create a News and Events Page
* Add the required fields and save
* A new section should appear below testimonial on your HomePage

#### Screenshots (if appropriate)
**Desktop**:
![image](https://user-images.githubusercontent.com/42243411/107488084-b62cd000-6ba8-11eb-8c35-8e2e230a0001.png)


**Mobile**:
![image](https://user-images.githubusercontent.com/42243411/107488110-bc22b100-6ba8-11eb-8a02-4e3ef15e08b3.png)
